### PR TITLE
feat: add automap

### PR DESCRIPTION
**Pull request by zezzipa**
### This PR:

(If this is a release PR, no need to add following. Leave this part empty)
(Use the following lines to create a PR text body. Make sure to remove all non-relevant one after you're done)
(Repeat each field as many times as necessary)

Added: for new features.
Changed: for changes in existing functionality.
Deprecated: for soon-to-be removed features.
Removed: for now removed features.
Fixed: for any bug fixes.
Security: in case of vulnerabilities.

### Review and tests: 
- [ ] Tests pass
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Code review
- [ ] `CHANGELOG.md` is updated
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

File: .tests/integration/config.yaml [hunk 1]
```diff
@@ -9,6 +9,10 @@ reference:
   design_intervals_gatk_cnv: "reference/twist_DNA_solid.HES45.annotated.interval_list"
   fasta: "reference/HES45.fna"
 
+automap:
```
Comment (jonca79) [inline .tests/integration/config.yaml:12]: Why no container here? No test data implemented?

File: workflow/envs/automap.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - bcftools=1.16
+  - bedtools=2.30.0
+  - perl=5.32.1
+# How to add R (>= v3.2.0) here?
```
Comment (Smeds) [inline workflow/envs/automap.yaml:9]:
```suggestion
  r-base=4.1.2
```

File: workflow/rules/automap.smk [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        threads=config.get("automap", {}).get("threads", config["default_resources"]["threads"]),
+        time=config.get("automap", {}).get("time", config["default_resources"]["time"]),
+    container:
+        config.get("automap", {}).get("container", config["default_container"])
+    conda:
+        "../envs/automap.yaml"
+    message:
+        "{rule}: Finding ROH regions cnv_sv/{rule}/{wildcards.sample}_{wildcards.type}.input"
+    shell:
+        "automap --vcf {input.vcf} --out $(dirname {output.dir}) --genome {params.build} {params.extra}"
```
Comment (Smeds) [inline workflow/rules/automap.smk:35]: we should try to create wrapper here, see issue #133.

Comment (jonca79) [inline workflow/rules/automap.smk:35]:
```suggestion
        "automap "
        "--vcf {input.vcf} "
        "--out $(dirname {output.dir}) "
        "--genome {params.build} "
        "{params.extra} &> {log}"
```

File: workflow/rules/automap.smk [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        mem_per_cpu=config.get("automap", {}).get("mem_per_cpu", config["default_resources"]["mem_per_cpu"]),
+        partition=config.get("automap", {}).get("partition", config["default_resources"]["partition"]),
+        threads=config.get("automap", {}).get("threads", config["default_resources"]["threads"]),
+        time=config.get("automap", {}).get("time", config["default_resources"]["time"]),
+    container:
+        config.get("automap", {}).get("container", config["default_container"])
+    conda:
+        "../envs/automap.yaml"
+    message:
+        "{rule}: Finding ROH regions cnv_sv/{rule}/{wildcards.sample}_{wildcards.type}.input"
```
Comment (jonca79) [inline workflow/rules/automap.smk:33]:
```suggestion
        "{rule}: Finding ROH regions {output.tsv}"
```

**Pull request merged**